src/onnx-mlir.cpp

```diff
@@ -58,10 +58,9 @@ int main(int argc, char *argv[]) {
 
   llvm::cl::SetVersionPrinter(getVersionPrinter);
 
-  // Parse options from argc/argv and default ONNX_MLIR_FLAG env var.
+  // Parse options from argc/argv and env var.
   llvm::cl::ParseCommandLineOptions(argc, argv,
-      "ONNX-MLIR modular optimizer driver\n", nullptr,
-      OnnxMlirEnvOptionName.c_str());
+      "ONNX-MLIR modular optimizer driver\n", nullptr, customEnvFlags.c_str());
```

Comment:
If inside the new C interface function named omCompileFromFileViaCommand, I give a name based on the input model file name before onnxmlirCompile.exec(), will this still be a potential problem? Actually from my understanding, since using the `onnx-mlir` exec as a target, it will work for multi-threading and multi-processing, if I use the onnxmlirCompile.exec() to run it, no matter what is the name comes from, right?